logfire/_internal/main.py

```diff
@@ -2041,7 +2025,7 @@ def end(self) -> None:
                         ATTRIBUTES_JSON_SCHEMA_KEY, attributes_json_schema(self._json_schema_properties)
                     )
 
-                self._span.end()
+            self._span.end(end_time)
```

Comment:
why is this dedented?

Reply:
not sure if it was intentional, if it was, I would guess it was because it's using the wrapped spans `.end()`, which isn't error-suppressed by otel and is for a span we didn't create, so this would preserve the error behavior. But anyway I can just re-indent it